python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -484,32 +484,32 @@ def do_evaluate(
             return self._distinct(
                 column,
                 keep=plc.stream_compaction.DuplicateKeepOption.KEEP_FIRST,
-                source_value=plc.interop.from_arrow(pa.scalar(True)),  # noqa: FBT003
-                target_value=plc.interop.from_arrow(pa.scalar(False)),  # noqa: FBT003
+                source_value=plc.interop.from_arrow(pa.scalar(True, type=pa.bool_())),  # noqa: FBT003
```

Comment:
question: Do we also know the type (from polars)?

If so, maybe we want to consider adding a ``pa_scalar`` helper that requires the type, and then just use the linter to ban uses of ``pa.scalar``

Reply:
The polars dtype does exist so we can use that.

Do you know of linters that can ban use of certain functions (I'd rather not use regular expressions)?

Reply:
I don't think there's any at the moment.

I think it's possible to specify a rule for `pa.scalar(blah)` using ast-grep at the very least, but that's not robust to cases where `scalar` is imported from `pyarrow` differently.

Reply:
I've used the polars dtype, but am not adding the lint: eventually we should have direct scalar factories in pylibcudf I think.